lib/enum/basic_enum.rb

```diff
@@ -22,48 +22,58 @@ def number_of_elements_in_collection
   # This one also uses 'count'
   # Try it yourself!
   def number_of_floats_or_fixnums
+    collection.select { |x| x.is_a? (Numeric)}.count
```

Comment:
Make sure you're naming your variables according to the thing they represent. In this case, maybe `|num|` or `|number|`.